pandas/tests/strings/test_case_justify.py

```diff
@@ -229,82 +254,105 @@ def test_center_ljust_rjust():
             np.nan,
         ]
     )
-    assert isinstance(rs, Series)
-    tm.assert_almost_equal(rs, xp)
+    tm.assert_series_equal(result, expected)
 
 
-def test_center_ljust_rjust_fillchar():
-    values = Series(["a", "bb", "cccc", "ddddd", "eeeeee"])
+def test_center_ljust_rjust_fillchar(any_string_dtype):
```

Comment:
These cases are different from the mixed object cases which is why they should not be split.

here, each dtype is tests explicitly against an expected. and then against the standard library.

the test that the expected matches the standard library really only to be done the once, but have changed the test so that there is not the 'special case' for object.